drop-transfer/src/event.rs

```diff
@@ -28,7 +28,7 @@ pub enum Event {
     FileUploadFailed(Transfer, FileId, Error),
     FileDownloadFailed(Transfer, FileId, Error),
 
-    TransferCanceled(Transfer, bool),
+    TransferCanceled(Transfer, bool, bool),
```

Comment:
two bools in a row is a very error-prone way. Let's introduce a struct here and use it as a second parameter instead of two bools

Reply:
fixed